datacube_alchemist/_utils.py

```diff
@@ -33,19 +29,24 @@ def compute(self, data) -> Dataset:
 
 
 def _get_logger(name: str):
-    logger = structlog.get_logger(name)
-    logging.basicConfig(
-        format="%(asctime)s %(name)s %(levelname)s %(message)s",
-        stream=sys.stdout,
-        level=logging.INFO,
-        datefmt="%y-%m-%d %H:%M:%S",
-    )
+    processors = [
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M:%S"),
+        structlog.processors.StackInfoRenderer(),
+        structlog.processors.format_exc_info,
+        structlog.dev.ConsoleRenderer(),
+    ]
+
     structlog.configure(
-        processors=[structlog.processors.KeyValueRenderer(key_order=["event"])],
-        context_class=structlog.threadlocal.wrap_dict(dict),
-        logger_factory=structlog.stdlib.LoggerFactory(),
+        processors=processors,
```

Comment:
This only needs to run once per application execution, I'd put it in `def cli()` in `worker.py`.

I don't think combining 'getting a logger', with 'setting up the logging framework' is a good idea.